assets/js/src/segments/list.jsx

```diff
@@ -233,6 +238,23 @@ const itemActions = [
       return !isSpecialSegment(segment) && segment.automated_emails_subjects.length > 0;
     },
   },
+  {
+    name: 'delete_wp_users',
+    className: 'mailpoet-hide-on-mobile',
+    label: MailPoet.I18n.t('trashAndDisable'),
+    onClick: function onClick(segment) {
+      MailPoet.Notice.error(
```

Comment:
Is the `error` really the best type of notice here? And also instead of the outdated `MailPoet.Notice`, we should use `assets/js/src/notices/notice.tsx`.

Reply:
I used the same what is on line 229. I didn't know that it's outdated.